### DISCUSSION TITLE
Resize window splits

### DISCUSSION THREAD

**Discussion by null-p01n73r** [helix-editor/helix#7812] [Troubleshooting]
Is it possible to resize the window splits explicitly, so that I can arbitrarily adjust the dimensionality of the splits in my current view?  For example, lets say I have java code in one window, and markdown in another:  I'd like to set the md window to 40 width (or a percent, or anything other than an even split), and the Java to claim the rest.  I'm not seeing anything in the docs that shows how to do this, but I'm pretty new and may not have bumped into it yet.

**Comment by the-mikedavis** [comment]
This isn't currently possible. See https://github.com/helix-editor/helix/issues/1176
